dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4177,3 +4177,21 @@ def test_deprecate_gather_statistics(tmp_path, engine):
             gather_statistics=True,
         )
     assert_eq(out, df)
+
+
+@pytest.mark.xfail
```

Comment:
xfailed since the `cudf` changes aren't merged yet. Will un-xfail after both these PRs are merged.

Reply:
Would it be worthwhile temporarily add the cudf dispatch definition here, and avoid the xfail?